### DISCUSSION TITLE
The new OrcaSlicer V1.8.1 - TREE supports

### DISCUSSION THREAD

**Discussion by MarioL77** [OrcaSlicer/OrcaSlicer#3175] [Q&A] [answered]
Hello, I have a question:
The new OrcaSlicer V1.8.1 does not have TREE supports?

**Comment by LJKraus** [comment, marked as answer]
Yes, it does, manual and auto in several styles.
![image](https://github.com/SoftFever/OrcaSlicer/assets/139707784/0702d84f-53c1-48c1-b6b6-db828cc3b5a3)

**Comment by Ryslik777** [comment]
thx
